app/lint.xml

```diff
@@ -13,5 +13,7 @@
         <ignore path="**/ic_person_24dp.png"/>
         <ignore path="**/ic_search_24dp.png"/>
     </issue>
-
+    <issue id="UnusedNamespace" severity="error"/>
```

Comment:
Thanks I included SdCardPath, NewerVersionAvailable and Typos issues. For others I think lint probably already raises errors on OnClick,MissingPermission,UnknownId,DuplicateIds and RequiredSize and warnings on PluralsCandidate, StaticFieldLeak.